### DISCUSSION TITLE
RC command

### DISCUSSION THREAD

**Discussion by and-sh** [iNavFlight/inav#10552] [Q&A]
Do I understand correctly that the rcCommand[] array is used simultaneously by the pilot and the autopilot. And some functions like angle-throttle-correction can correct it for the pilot.

**Comment by sensei-hacker** [comment]
rcCommand is generally an input to all of the processing.

On fixed wing, it is set by code in modes where INAV is controlling the altitude.

**Comment by and-sh** [comment]
It is true that, the rcCommand[] is also displayed on the sticks? That is, the multirotor can completely controls the rcCommand[].
![Screenshot From 2024-12-25 08-02-11](https://github.com/user-attachments/assets/1542b83d-6a30-44b8-a804-b9d5f53828b8)

How to guess where the data comes from the pilot and where from the autopilot? If I change the rcCommand[] to rxGetChannelValue() in some function, will the autopilot stop working?
